docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
I'm tempted to remove this file and not support windows

Reply:
got it, I will remove the .bat file now